readme.md

```diff
@@ -114,10 +114,12 @@ const bot = new Wechaty({
 })
 
 bot
-  .on('scan', (qrcode) => {
-    QrcodeTerminal.generate(qrcode, {
-      small: true
-    })
+  .on('scan', (qrcode, status) => {
+    if (status === 2) {
```

Comment:
Why not use enum here?

Reply:
It need import from `wechaty-puppet`, so it will let user to install `wechaty-puppet`.

Reply:
`wechaty-puppet` is a dependency of `wechaty` and `wechaty-puppet-padplus`, which will be automatically installed.